installer/templates/phx_assets/package.json

```diff
@@ -10,21 +8,20 @@
     "phoenix": "file:<%= phoenix_webpack_path %>"<%= if html do %>,
     "phoenix_html": "file:<%= phoenix_html_webpack_path %>"<% end %><%= if live do %>,
     "phoenix_live_view": "file:<%= phoenix_live_view_webpack_path %>",
-    "nprogress": "^0.2.0"<% end %>
+    "nprogress": "^0.2"<% end %>
   },
   "devDependencies": {
-    "@babel/core": "^7.0.0",
-    "@babel/preset-env": "^7.0.0",
-    "babel-loader": "^8.0.0",
-    "copy-webpack-plugin": "^5.1.1",
-    "css-loader": "^3.4.2",
-    "sass-loader": "^8.0.2",
-    "sass": "^1.27.0",
-    "hard-source-webpack-plugin": "^0.13.1",
-    "mini-css-extract-plugin": "^0.9.0",
-    "optimize-css-assets-webpack-plugin": "^5.0.1",
-    "terser-webpack-plugin": "^2.3.2",
-    "webpack": "4.41.5",
-    "webpack-cli": "^3.3.2"
+    "@babel/core": "^7.x",
+    "@babel/preset-env": "^7.x",
+    "babel-loader": "^8.x",
+    "copy-webpack-plugin": "^6.x",
+    "css-loader": "^5.x",
+    "css-minimizer-webpack-plugin": "^1.x",
+    "mini-css-extract-plugin": "^1.x",
+    "sass": "^1.x",
+    "sass-loader": "^10.x",
+    "terser-webpack-plugin": "^5.x",
```

Comment:
Noticed on there [plugin page for Terser](https://webpack.js.org/plugins/terser-webpack-plugin/) it mentions:

> If you are using webpack v5 or above you do not need to install this plugin. Webpack v5 comes with the latest `terser-webpack-plugin` out of the box. 

So I think that means we can remove this line too? 🎉

Reply:
Thanks, wasn't aware of that.
It works really as expected without it.

Reply:
@cw789 did you uninstall and removed it from the plugins list? Of still need to require and keep it in the plugins?
The docs are rather vague.

Reply:
I re-added it to the `package.json`, as we directly require it in the `webpack.config.js`.
It would also work with not having it in the `package.json` but that kind of seems wrong to me.

Reply:
The docs kinda gave me the impression it is bundled with webpack.
The same was done with [ProvidePlugin](https://webpack.js.org/plugins/provide-plugin/) which can be used as:

```js
const webpack = require("webpack");
new webpack.ProvidePlugin({ ... }),
```

Reply:
Ahh. That's a different thing.
So we should remove it from `package.json` and in `webpack.config.js` - `require("webpack")` and use it as `new webpack.TerserPlugin`...

Reply:
No my point was I hoped this would work but it doesn't :(

Reply:
Looking at this the TerserPlugin is added by default:
https://webpack.js.org/configuration/mode/#mode-production (see plugins).

So adding `new TerserPlugin({})` might not be needed.

Reply:
Seems to work only if i completely remove the `optimization` part.
But then we don't have minified CSS...

Reply:
It'll work if I replace `new TerserPlugin({})` with `'...',`.
But that seems a bit weird to have a string escaped JS spread syntax there.

It's documented here: https://webpack.js.org/plugins/mini-css-extract-plugin/#minimizing-for-production
(but without the `,` at the end it's not working on my side)

And here: https://webpack.js.org/blog/2020-10-10-webpack-5-release/#changes-to-the-configuration

Reply:
If we use 
```
minimizer: [`...`, new CssMinimizerPlugin()]
```
We can remove both
```
"terser-webpack-plugin": "^5.x",
```
and
```
const TerserPlugin = require('terser-webpack-plugin');
```
from `package.json` and `webpack.config.js` respectively.